templates/publisher/account-snaps.html

```diff
@@ -25,7 +25,7 @@ <h2 class="p-heading--four">Get started&hellip;</h2>
       <div class="col-6">
         <h3 class="p-heading--five">'Hello world' with snapcraft</h3>
         <p>Try this 15 minute tutorial to get to know the basics of the snap format and snapcraft tool.</p>
-        <p><a href="https://tutorials.ubuntu.com/tutorial/create-your-first-snap#0" class="p-button--neutral p-link--external">Try the tutorial</a></p>
+        <p><a href="https://tutorials.ubuntu.com/tutorial/create-your-first-snap#0" target="_blank" class="p-button--neutral p-link--external">Try the tutorial</a></p>
```

Comment:
It's a good security practice to add  `rel="noreferrer noopener"` as well:
https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/

Although in this case we fully control this link and it's our site, so it's not a deal breaker.

Reply:
definitely worth adding though, thanks @bartaz